tests/core/test_model_manager.py

```diff
@@ -616,6 +618,60 @@ def test_find_without_on_disk_model(good_model_path, reset_globals):
         assert not os.path.exists("some_pretend_model")
 
 
+def test_find_without_on_disk_model_and_load_path(good_model_path, reset_globals):
+    """Make sure that ephemeral models can be found without existing on disk
+    when a shared load_path is configured
+    """
+    with tempfile.TemporaryDirectory() as workdir:
+        with temp_config(
+            {
+                "load_path": workdir,
+                "model_management": {
+                    "finders": {
+                        "default": {
+                            "type": TestFinder.name,
+                            "config": {
+                                "fallback_to_local": False,
```

Comment:
`fallback_to_local` does't seem to get activated since https://github.com/caikit/caikit/pull/425/files#diff-b6b4baaaf129f8633d11eaae2a9b6ae490f04e65276cc1d541cafe822422e31fR90 will be false?

Reply:
Yep! Totally missed that. Good catch